src/main/java/org/spongepowered/mod/text/SpongeTextFactory.java

```diff
@@ -54,6 +55,10 @@ public Text parseJson(String json) throws IllegalArgumentException {
         }
     }
 
+    public static Locale getDefaultLocale() { // TODO: Get this from the MC client?
+        return Locale.getDefault();
```

Comment:
yeah, the server's default locale is in a separate option so that something like that can be done without a ton of difficulty.